tests/FSharp.Compiler.ComponentTests/Signatures/TestCasesForGenerationRoundTrip/hash_constraint_seq.fsx

```diff
@@ -0,0 +1,7 @@
+module Foo
+
+open System
+open System.Threading.Tasks
+
+let mapWithAdditionalDependenies
+    (mapping: 'a -> 'b * #seq<#IDisposable>) = 0
```

Comment:
@T-Gro this is the test you are looking for. Without the parentheses, this will fail.
See `|> withOptions [ "--warnaserror:64" ]` change.